### DISCUSSION TITLE
Syncing theme with my webview

### DISCUSSION THREAD

**Discussion by kal-el11** [microsoft/vscode-discussions#781] [Extension Development QnA] [answered]
I have a web view with a monaco editor. I would like to sync the theme used by the vscode with my editor. Or Is it possible to reuse the editor provided by vscode in my web view.

**Comment by HatemMn** [comment]
Did you check [vscode-webview-ui-toolkit](https://github.com/microsoft/vscode-webview-ui-toolkit/tree/main) ?
I hope this is what you were looking for :)

**Comment by tamuratak** [comment, marked as answer]
See https://code.visualstudio.com/api/extension-guides/webview#theming-webview-content

> Webviews can also access VS Code theme colors using [CSS variables](https://developer.mozilla.org/docs/Web/CSS/Using_CSS_variables).

**Comment by olguzzar** [comment]
Hi @kal-el11, did either of the answers help solve your problem? If so, please mark it as answered. Otherwise, please provide more information so we can further assist!
